help/C/Help_ch_Transactions.xml

```diff
@@ -1067,28 +1067,112 @@ Author:
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry>Flagged for input-transfer account must be selected.</entry></row>
+                <row><entry>Flagged for input-transfer account matched.</entry></row>
+                </tbody>
+            </entrytbl>
+          </row>
+          <row>
+           <entrytbl  cols='2' align='left' colsep='1' rowsep='1'>
+                <tbody>
+                  <row><entry><emphasis role="strong">A    </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">U+R</emphasis></entry><entry><emphasis>X</emphasis></entry></row>
+                  <row><entry><emphasis role="strong">R    </emphasis></entry><entry>-</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry><?dbhtml bgcolor="#CCFFCC"?><?dbfo bgcolor="##9FFCC"?>Light Green</entry></row>
+                <row><entry><?dbhtml bgcolor="#FF3333"?><?dbfo bgcolor="#FF3333"?>Bright Red</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry>Matches an existing transaction - import and update amount.</entry></row>
+                <row><entry>Transaction not matched - not to be imported.</entry></row> <!-- FIXME might not have this quite correct -->
+                </tbody>
+            </entrytbl>
+          </row>
+          <row>
+           <entrytbl  cols='2' align='left' colsep='1' rowsep='1'>
+                <tbody>
+                  <row><entry><emphasis role="strong">A    </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">U+R  </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">R</emphasis></entry><entry><emphasis>X</emphasis></entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry><?dbhtml bgcolor="#CCFFCC"?><?dbfo bgcolor="#CCFFCC"?>Light Green</entry></row>
+                <row><entry><?dbhtml bgcolor="#FF3333"?><?dbfo bgcolor="#FF3333"?>Bright Red</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry>Matches an existing transaction - not to be imported.</entry></row>
+                <row><entry>Matches but not reconciled - not to be imported.</entry></row><!--FIXME might not have this quite correct -->
+                </tbody>
+            </entrytbl>
+          </row>
+          <row><entry></entry><entry><?dbhtml bgcolor="#6699CC"?><?dbfo bgcolor="#6699CC"?>Cornflower Blue</entry><entry>Row is selected</entry>
+          </row>
+        </tbody>
+        </tgroup>
+       </table>
```

Comment:
![Table match_status in yelp 3.32.2](https://user-images.githubusercontent.com/6572538/61819056-8a28ed00-ae52-11e9-8725-9c57bf1e9f93.png)
I believe, you intended something different?

Reply:
@fellen Certainly not what I intended. Yelp seems to handle the nesting of tables differently from HTML. I'll check through the various other output formats and see if I can get a form of the table which comes up correctly in all of them. I should be able to specify colum widths to get the header titles to match up with the columns in the nested tables.

Reply:
@fellen  The problem with the yelp viewer is an inability to render docbooks properly. The docbook sytax is correct and renders correctly in html and epub and mobi formats. The pdf output almost renders the tables correctly apart from a few slight problems. Yelp (3.26.0) is supposed to be able to view docbook format documents and/or the html documents. It doesn't read the html documents at all (gives a URL can't be shown error) and doesn't render the tables when started with the docbook format.  It may be that it doesn't find the gnc- docbookx.dtd.
 I was unable to view the docbook using the XMLMInd editor as a viewer either. It may also need links to the xsl stylesheets Not sure how we can fix it unless we can get the Gnome developers to fix yelp, It seems silly to restrict the other document formats for the sake of complying with  a poor viewer.  yelp's limitations seem to get a fair bit of airplay on various forums. I can raise the problem on the Yelp bugs list and see if that raises any response but it won't help in the short term.

David

Reply:
Might the problem with Yelp be due to a DocBook version difference?

Reply:
> Might the problem with Yelp be due to a DocBook version difference?

In /usr/share/yelp/dtd/catalog they have db 4.1.2 - 4.5 dtd's

Reply:
I can't find any reference to what version of DocBooks yelp supports but it seems to be both 4.5 and 5.0. Nested tables has been supported since v4.4 in DocBooks.

Reply:
I can't find any reference to what version of DocBooks yelp supports but it seems to be both 4.5 and 5.0. Nested tables has been supported since v4.4 in DocBooks.

Reply:
I don't really know enough about how docbooks works. I'll play with a dummy table in a dummy document and see what I can make work in yelp. I've put a question up about yelp and docbook on an ubuntu forum to see what comes back and I'll raise an issue about it on the main yelp gitlab bug reporting. My guess is they have only implemented a subset of the docbook capability sufficient for writing help type files. One of the reasons for the shift to mallard in yelp seems to be it is better suited for constructing help which is task oriented and it doesn't tag everything so it is more tolerant.

Reply:
The "normal" db2html processing of `<entrytbl>` is done in https://github.com/Gnucash/gnucash-docs/blob/maint/xsl/1.79.2/html/table.xsl#L696, but yelp and khelpcenter use their own xsl's. So I think, you should file a bug against yelp.

Reply:
Note that we don't generate those stylesheets, they come from [DocBook's repository](https://github.com/docbook/xslt10-stylesheets/).

Reply:
@jralls @fellen I will raise a bug report with yelp. Can't really see why they would use their own xsl's when DocBook make them available from their repository.  I have reworked the table so that it is at least presentable in yelp but so far it is problematical in HTML now.  I have set myself up properly to build out of source with a short script to run after I have pulled from GnuCash/gnucash-docs on github.